src/Workspaces/Core/Portable/Simplification/SimplificationOptions.cs

```diff
@@ -45,11 +46,32 @@ public static class SimplificationOptions
         public static Option<bool> AllowSimplificationToBaseType { get; } = new Option<bool>(NonPerLanguageFeatureName, "AllowSimplificationToBaseType", true);
 
         /// <summary>
-        /// This option says if we should simplify away the this. or Me. in member access expression
+        /// This option says if we should simplify away the <see langword="this"/>. or <see langword="Me"/>. in member access expressions.
```

Comment:
If you're going to make this less terrible, we probably should remove the . after this so it's grammatically correct.